tests/test_core.py

```diff
@@ -7256,7 +7260,8 @@ def setUp(self):
 binaryen3jo = make_run("binaryen3jo", compiler=CLANG, emcc_args=['-O3', '-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="interpret-binary,asmjs"'])
 
 # This only works when .emscripten specifies a JS_ENGINE with native wasm support.
-binaryen_native = make_run("binaryen_native", compiler=CLANG, emcc_args=['-O2', '-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="native-wasm"'])
+binaryen0_native = make_run("binaryen0_native", compiler=CLANG, emcc_args=['-O0', '-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="native-wasm"'])
```

Comment:
Yeah, binaryen test suite tests all the binaryen methods (interpret-s-expr, -binary, etc.), on a bunch of testcases. Not as comprehensive as the emscripten test suite, but that + that the interpreter itself is tested on the spec test suite, gives me confidence.